interface/forms/newpatient/common.php

```diff
@@ -183,7 +183,13 @@ class="css_button link_submit" target='Main' onClick="top.restoreSession()">
  </div>
 
 <br> <br>
-
+<?php
+if (!$viewmode) {
+    $query = "SELECT pc_aid FROM openemr_postcalendar_events WHERE pc_eventDate = ? and pc_pid =?";
+    $sched_visit = sqlStatement ($query, array($_SESSION['lastcaldate'],$pid) );
+    $result = sqlFetchArray($sched_visit);
```

Comment:
Seems wrong to use the name $result here, as that variable is used elsewhere to hold a form_encounter row.